eevee/metrics/classification.py

```diff
@@ -17,6 +18,12 @@ def intent_report(
     # for cases where we are seeing NaN values popping up.
     df[['intent_x', 'intent_y']] = df[['intent_x', 'intent_y']].fillna(value="_")
 
+    # aliasing intents
+    if intent_aliases is not None:
+        alias_dict = {intent: alias for alias, intent_list in intent_aliases.items() for intent in intent_list}
+        for col in ["intent_x", "intent_y"]:
+            df[col] = df[col].apply(lambda intent: alias_dict.get(intent, intent))
```

Comment:
`alias_dict.get(intent, intent)` modifies `intent` if it is a key in the dictionary, else keeps it unchanged